CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -61,10 +72,22 @@ public void setArmMotorPower(double power) {
         rightArmMotor.set(power);
     }
     
+    public void forceCalibrateLow() {
+        armEncoderCalibrationHeight.set(getArmAngle());
+        armEncoderCalibrated.set(true);
+    }
+    
+    public boolean isCalibrated() {
+        return armEncoderCalibrated.get();
+    }
+    
     public void updateSensors() {
         armAngleDegrees.set(getArmAngle());
         lowerLimitSwitchProperty.set(lowerLimitSwitch.get());
         upperLimitSwitchProperty.set(upperLimitSwitch.get());
         
+        if (lowerLimitSwitch.get()) {
```

Comment:
It feels a little sketchy for this method to have functional side effects, but I don't have a better idea offhand for where to put this logic.